src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -108,6 +108,21 @@ class JooqTouchstoneRepository(
         return ScenarioAndCoverageSets(scenario, getCoverageSetsFromRecord(records, scenario))
     }
 
+    override fun getTouchstonesByGroupId(groupId: String): List<Touchstone>
+    {
+        var query = dsl
```

Comment:
Here we first need to dereference the group id, and throw an error if the group does not exist. This can all be done by the method `modellingGroupRepository.getModellingGroup(groupId)`, see the method `getModelRunParamaterSets` in the `JooqBurdenEstimateRepository` as an example of this.

We should also add a test for this (see `exception is thrown for non-existent modelling group ID` in `GetModellingGroupTests` for an example of checking for an UnknownObjectError)